### ISSUE TITLE
RuntimeError: Constants DL and Fiddle is not defined.

### ISSUE THREAD

**Issue by Arakaki** [arton/rjb#70]
I report RuntimeError here.
Might need to fix to support JDK version 11 or more.
* error log
```
RuntimeError:
  Constants DL and Fiddle is not defined.
...
# Fiddle::DLError:
#   /usr/lib/jvm/java-11-openjdk-amd64/jre/lib/amd64/server/libjvm.so: cannot open shared object file: No such file or directory
```

* cause
JDK 11+ do not include jre so can not open libjvm.so file.
https://github.com/arton/rjb/blob/faee51fad1a4f7cf608bfe0485d8cc244b6a6991/ext/load.c#L90

* temporary solution
``` bash
ln -s /usr/lib/jvm/java-11-openjdk-amd64/lib/server /usr/lib/jvm/java-11-openjdk-amd64/jre/lib/amd64/server
```

* environments
  * JDK: JDK11
  * Ryby: 2.5.5
  * OS: Linux

**Comment by Arakaki** [comment]
Also might be related to #69

**Comment by wynksaiddestroy** [comment]
Is there anyone who has a clue about how to fix this issue?

**Comment by arton** [comment]
I fixed this issue by 1.6.2.
